collector/filesystem_freebsd.go

```diff
@@ -21,78 +21,65 @@ import (
 import "C"
 
 const (
-	filesystemSubsystem = "filesystem"
+	subsystem = "filesystem"
```

Comment:
This is in package scope. So it's confusing for all the other collectors if you pick a very generic name here.